src/apps/bridge/reportBuilder.cpp

```diff
@@ -464,6 +495,76 @@ static bool addSamplesToReport(sensor_report_encoder_context_t &context, uint8_t
     rval = true;
     break;
   }
+  case SENSOR_TYPE_RBR_CODA: {
+    rbr_coda_aggregations_t rbr_coda_sample =
+        (static_cast<rbr_coda_aggregations_t *>(sensor_data))[sample_index];
+    bool rbr_coda_sample_valid = false;
+    switch (rbr_coda_sample.sensor_type) {
```

Comment:
switch statement to encode the correct rbr type. Spotter will use this to filter out the sample_members since we will always send all three; temp_mean, pressure_mean, pressure_stdev.